interpreter.asm

```diff
@@ -76,14 +107,14 @@ quit_reset
     stx     SOURCE_ID_MSB
     stx     SAVE_INPUT_STACK_DEPTH
     stx     READ_EOF
-    jsr     $ffcc   ; CLRCHN
+    jsr $ffe7       ; CLALL. Clear file table; call CLRCHN.
```

Comment:
This use of CLALL might cause problems, as it does not CLOSE any open files.

Reply:
Okay, CLRCHN it is.  Thanks for having my back on this.  :)